### PR TITLE
ENCD-5467 Update Genetic Modification UI for new property

### PR DESCRIPTION
**Pull request by forresttanaka**

### CODE REVIEW

**Review by zoldello — DISMISSED** [review]
Changes look good. I have nothing to add.

**Comment by forresttanaka** [comment]
For second code review: moved the method out of the search results link and into up to two lines below the link, above the Lab line.

File: src/encoded/static/components/genetic_modification.js [hunk 1]
```diff
@@ -673,9 +682,13 @@ const ListingComponent = (props, reactContext) => {
         <li className={resultItemClass(result)}>
             <div className="result-item">
                 <div className="result-item__data">
-                    <a href={result['@id']} className="result-item__link">{result.category} &mdash; {result.purpose} &mdash; {result.method}</a>
+                    <a href={result['@id']} className="result-item__link">{result.category} &mdash; {result.purpose}</a>
                     <div className="result-item__data-row">
                         {result.modified_site_by_target_id ? <div><strong>Target: </strong>{result.modified_site_by_target_id.name}</div> : null}
+                        {result.method ? <div><strong>Method: </strong>{result.method}</div> : null}
```
Comment (zoldello) [inline src/encoded/static/components/genetic_modification.js:688]: @forresttanaka  This is rhetorical/philosophical - should strong tag be used or should the bold be added via CSS?

File: src/encoded/static/components/genetic_modification.js [hunk 2]
```diff
@@ -735,6 +748,10 @@ GeneticModificationSummary.columns = {
     category: { title: 'Category' },
     purpose: { title: 'Purpose' },
     method: { title: 'Method' },
+    nucleic_acid_delivery_method: {
+        title: 'Nucleic acid delivery method',
+        display: item => (item.nucleic_acid_delivery_method && item.nucleic_acid_delivery_method.length > 0 ? item.nucleic_acid_delivery_method.join(', ') : null),
```
Comment (zoldello) [inline src/encoded/static/components/genetic_modification.js:753]: @forresttanaka 
[].join(', ') results in- ''.  So, is "item.nucleic_acid_delivery_method.length > 0" needed?

**Review by zoldello — APPROVED** [review]
looks good. My comments are not significant.

**Pull request merged**